README.md

```diff
@@ -140,26 +140,26 @@ A reducer that keeps track of Router state.
 
 A component that renders a React Router app using router state from a Redux store.
 
-### `pushState(state, pathname, query)`
+### `push(state, pathname, query)`
 
-An action creator for `history.pushState()`. (https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)
+An action creator for `history.push()`. (https://developer.mozilla.org/en-US/docs/Web/API/History/pushState)
```

Comment:
Is this a reference to the original pushState()-method? If yes, then this change is wrong.